napari/plugins/tests/test_hookspecs.py

```diff
@@ -0,0 +1,87 @@
+import inspect
+import warnings
+
+import pytest
+from numpydoc.docscrape import FunctionDoc
+
+from napari.plugins import hookspecs
```

Comment:
... I'd like this to read `from napari.plugins import hook_specifications`..., and below, I'd probably go for a list comprehension:

```python
HOOK_SPECIFICATIONS = [
    name, func for name, func in vars(hook_specifications).items() if hasattr(func, 'napari_spec')
]

Reply:
changed the name of the module